CONTRIBUTING.md

```diff
@@ -1,35 +1,36 @@
 How to report a bug
 -------------------
 - Use the search function to see if someone else has already reported the bug. If an issue has already been opened, see if you can add any useful information to it.
-- Otherwise, create a new issue on the "issues" page
-- Describe the problem as detailed as possible.
-- Some bugs only occur on certain devices or versions of Android. Please add information about your device and the version of Android that is running on it (you can look these up under `Settings → About Phone`) as well as which version of AntennaPod you are using.
-- If the bug only seems to occur with a certain podcast, make sure to include the URL of that podcast.
+- Otherwise, create a new issue on the "issues" page.
+- Describe the problem with as much detail as possible.
```

Comment:
Not a native speaker but I feel like the old one sounds better (also, it's shorter)

Reply:
"as detailed" works, but "with as much detail" is more grammatically formal.